scripts/pre_commit_hook.py

```diff
@@ -63,8 +63,7 @@
     b'FIREBASE_CONFIG_APP_ID', b'FIREBASE_CONFIG_AUTH_DOMAIN',
     b'FIREBASE_CONFIG_MESSAGING_SENDER_ID', b'FIREBASE_CONFIG_PROJECT_ID',
     b'FIREBASE_CONFIG_STORAGE_BUCKET', b'FIREBASE_CONFIG_GOOGLE_CLIENT_ID']
-NPX_PATH: Final = os.path.join(
-    os.pardir, 'oppia_tools', 'node-16.13.0', 'bin', 'npx')
+NPX_PATH: Final = 'npx'
```

Comment:
I don't think this should be committed.

Reply:
Yup, was in my radar, just waited if any other changes are required. Reverted. Thanks!